gdtf.py

```diff
@@ -131,6 +131,24 @@ def loadPrimitive(model):
         obj.scale = (model.length/obj.dimensions.x,model.width/obj.dimensions.y,model.height/obj.dimensions.z)
         return obj
 
+    @staticmethod
+    def loadglb(profile, model):
```

Comment:
One thing I've noticed after merging: this method should follow the lowerCamelCase standard, so it should be `loadGlb`.
Could you fix this in a future PR?